### DISCUSSION TITLE
How to change color of loading image?

### DISCUSSION THREAD

**Discussion by elenajobelena** [HC200ok/vue3-easy-data-table#168] [Q&A] [answered]
How to change color of loading image?
I have tried 
`<style>  
  .lds-ring div[data-v-1fa3a520] {
    border-color: #E4F1F7;
  }  
</style>`

**Comment by HC200ok** [comment, marked as answer]
You can try theme color feature: https://hc200ok.github.io/vue3-easy-data-table-doc/features/theme-color.html

**Reply by elenajobelena** [reply]
Thank you for a link: https://codesandbox.io/s/theme-color-89ps64?file=/src/App.vue:1755-1784
It works.
